src/status_im/translations/fr.cljs

```diff
@@ -182,52 +182,35 @@
    :remove                                "Retirer"
    :save                                  "Sauvegarder"
    :delete                                "Supprimer"
-   :change-color                          "Changer la couleur"
    :clear-history                         "Effacer l'historique"
    :mute-notifications                    "Couper le son des notifications"
    :leave-chat                            "Quitter le chat"
-   :delete-and-leave                      "Supprimer et quitter"
    :chat-settings                         "Paramètres de chat"
    :edit                                  "Modifier"
    :add-members                           "Ajouter des membres"
-   :blue                                  "Bleu"
-   :purple                                "Violet"
-   :green                                 "Vert"
-   :red                                   "Rouge"
 
    ;commands
-   :money-command-description             "Envoyer de l'argent"
-   :location-command-description          "Envoyer l'emplacement"
-   :phone-command-description             "Envoyer le numéro de téléphone"
-   :phone-request-text                    "Demande de numéro de téléphone"
-   :confirmation-code-command-description "Envoyer le code de confirmation"
-   :confirmation-code-request-text        "Demande de code de confirmation"
-   :send-command-description              "Envoyer l'emplacement"
-   :request-command-description           "Envoyer une demande"
-   :keypair-password-command-description  ""
-   :help-command-description              "Aide"
-   :request                               "Demande"
    :chat-send-eth                         "{{amount}} ETH"
-   :chat-send-eth-to                      "{{amount}} ETH à {{chat-name}}"
-   :chat-send-eth-from                    "{{amount}} ETH de {{chat-name}}"
+   
+   ;;location command
+   :your-current-location                 "Votre localisation actuelle"
+   :places-nearby                         "Endroits proches"
+   :search-results                        "Résultats de recherche"
+   :dropped-pin                           "Goupille placée"
```

Comment:
`Goupille` sounds weird?

Reply:
I agree, I'll replace it with `repère` does it sounds better?